serverlessworkflow/modules/ROOT/pages/index.adoc

```diff
@@ -248,6 +248,14 @@ xref:persistence/persistence-with-postgresql.adoc[Running a workflow service usi
 Running Serverless Workflow service using PostgresSQL
 --
 
+[.card]
+--
+[.card-title]
+xref:persistence/postgresql-flyway-migration.adoc[Migrate Your Postgresql database with Kogito upgrade]
+[.card-description]
+Migrate your existing database with changes from Kogito upgrade
```

Comment:
If this is a procedure module then add gerund ( the ing form t the end of the verb). Just a suggestion.

```suggestion
Migrating your existing database with changes from the Kogito upgrade
```